src/unraphael/dash/pages/4_compare.py

```diff
@@ -120,6 +125,44 @@ def align_images_widget(*, base_image: ImageType, images: list[ImageType]) -> li
     return res
 
 
+def display_two(base_image, images):
+    col1, col2 = st.columns(2)
+    
+    if 'count' not in st.session_state:
+        st.session_state.count = 0
+
+    if 'images' not in st.session_state:
+        st.session_state.images = images  # Directly assign the list
+
+    def display_image():
+        try:
+            image = st.session_state.images[st.session_state.count].data
+            col1.image(base_image, caption="Base Image", use_column_width=True)
+            col2.image(image, caption=f"Image {st.session_state.count + 1}", use_column_width=True)
+        except IndexError as e:
+            st.error(f"Error displaying image: {e}")
```

Comment:
Take non-failing code out of the try statement

```suggestion
        try:
            image = st.session_state.images[st.session_state.count].data
        except IndexError as e:
            st.error(f"Error displaying image: {e}")
        else:
            col1.image(base_image, caption="Base Image", use_column_width=True)
            col2.image(image, caption=f"Image {st.session_state.count + 1}", use_column_width=True)
```

Reply:
done